californium-core/src/main/java/org/eclipse/californium/core/server/resources/ResourceAttributes.java

```diff
@@ -251,6 +251,31 @@ public void clearObservable() {
 		attributes.remove(LinkFormat.OBSERVABLE);
 	}
 
+	/**
+	 * Marks the resource as only accessible using OSCORE.
+	 */
```

Comment:
Please add:

```
@since 3.10
```

Reply:
Done in commit e41bd07e2f4956c067eb4774a2edcee45e7e7a3a